openshift/scripts/database-dump.sh

```diff
@@ -76,7 +76,7 @@ SAVE_PATH=$( dirname ${SAVE_TO} )
 mkdir -p ${SAVE_PATH}
 oc exec ${POD_DB} -n ${PROJECT} -- /bin/bash -c \
 	'pg_dump -U ${POSTGRESQL_USER} -d ${POSTGRESQL_DATABASE} -Fc \
-	--column-inserts --data-only -T mms.* -T public.etl* -T public.flyway* > /tmp/'${SAVE_FILE}
+	--column-inserts --data-only -T mms.* -T public.etl* -T nris.* -T public.flyway* > /tmp/'${SAVE_FILE}
```

Comment:
Thanks for adding this, it solves a permission issue and we don't want to back it up anyhow.